### DISCUSSION TITLE
Is there any point to using Privacy Badger with an existing adblocker like uBlock Origin?

### DISCUSSION THREAD

**Discussion by Milo123459** [EFForg/privacybadger#2841] [Q&A] [answered]
I installed the extension a few days ago and have continued browsing normally, but it has caught no trackers. I believe my adblocker is blocking all of them, so I'm curious - is it worth installing and running whilst an adblocker is also running?

**Comment by ghostwords** [comment, marked as answer]
Yes, see the following Privacy Badger FAQ entry:

[privacybadger.org > FAQs > Is Privacy Badger compatible with other extensions, including adblockers?](https://privacybadger.org/#Is-Privacy-Badger-compatible-with-other-extensions%2c-including-other-adblockers)

Let me know if you want me to go into any of the points in the FAQ in more detail.

**Reply by Milo123459** [reply]
Thanks for the information! Is there any way to see how many trackers Privacy Badger has blocked?

**Reply by ghostwords** [reply]
To see how many domains Privacy Badger ...

- blocked on a site: Open Privacy Badger's popup by clicking on Privacy Badger's icon in your browser toolbar.
- has learned to block in total: Open Privacy Badger's options page and select the Tracking Domains tab. There will be a summary at the top.

**Reply by Milo123459** [reply]
Thank you very much!
